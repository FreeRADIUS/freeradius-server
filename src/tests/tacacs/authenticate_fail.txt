#
#	ARGV: -t pap -u scald authenticate -p pegapilha
#