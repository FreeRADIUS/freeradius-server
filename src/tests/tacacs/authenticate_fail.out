status: FAIL
data: b'Authentication-Data'
server_msg: b'Authentication-Start failed for scald'
